lib/beamchmark/formatters/html.ex

```
@@ -69,10 +72,17 @@ defmodule Beamchmark.Formatters.HTML do
   end
 
   defp get_browser() do
-    case :os.type() do
-      {:unix, :darwin} -> "open"
-      {:unix, _} -> "xdg-open"
-      {:win32, _} -> "explorer"
+    case Utils.os() do
+      :macOS -> "open"
+      :Windows -> "explorer"
+      :Linux -> "xdg-open"
+    end
```

There's no case clause for FreeBSD - this case will throw an error. 

AFAIK `xdg-open` is available on the FreeBSD, and probably whole beamchmark will work on it. But, if you don't want to support it, maybe you should throw the custom error with explanation, that this system is not supported.

Also, I think it's a good idea to describe which systems are supported in the README file.